rackcity/views/log_views.py

```diff
@@ -49,10 +50,12 @@ def log_many(request):
     except Exception as error:
         return JsonResponse(
             {"failure_message": "Filter error: " + str(error)},
-            status=HTTPStatus.BAD_REQUEST
-        )
-    for filter_arg in filter_args:
-        logs = logs.filter(**filter_arg)
+            status=HTTPStatus.BAD_REQUEST)
+    if len(filter_args) > 0:
+        q_objects = Q()
+        for filter_arg in filter_args:
+            q_objects |= Q(**filter_arg)
+        logs = logs.filter(q_objects)
```

Comment:
REALLY TRICKY BUSINESS had to figure out how to apply filters as ORs since we're checking if username OR asset hostname OR asset number fits the search query the user asked for. Q objects are fun